### DISCUSSION TITLE
JIT compiler from scratch – 2/3

### DISCUSSION THREAD

**Discussion by giscus** [srijan-paul/srijan-paul.github.io#18] [Announcements]
# JIT compiler from scratch – 2/3

Exploring the Arm architecture in part 2 of the JIT compilation series

https://injuly.in/blog/jit-02/index.html

**Comment by janbaig** [comment]
Great write up. Looking forward to part 3!

**Reply by srijan-paul** [reply]
Thank you!
Part 3 is out now: https://injuly.in/blog/jit-03/
